middleware/auth.ts

```diff
@@ -6,4 +6,12 @@ export default defineNuxtMiddleware((context) => {
    * @todo Redirect the user to main page if token is present in store
    * @hints check what propeties context has
    */
+
+  let x = context.store.getters
+  //console.log(x)
+
+  if(!x.auth) {
+    context.redirect('/login/');
+  }
```

Comment:
if user is logged it then he should be prevented from going to login and register too

Reply:
how do  I do that? I tried to put an else statement in there and redirect to root but its giving an error "redundant navigation"

Reply:
i will post in a ideal soln soon